using_images/db_images/mongodb.adoc

```diff
@@ -76,31 +76,53 @@ $ oc new-app -e \
 
 === Running MongoDB Commands in Containers
 
-OpenShift uses https://www.softwarecollections.org/[Software Collections] to
-install and launch MongoDB. If you want to execute a MongoDB command inside of a
-running container (for debugging), you must invoke it using bash, to make sure
-the MongoDB collection is enabled, for example:
+OpenShift uses https://www.softwarecollections.org/[Software Collections] (SCLs)
+to install and launch MongoDB. If you want to execute MongoDB command inside of
+a running container (for debugging), you must invoke it using bash.
+
+To do so, first identify the name of the running MongoDB pod. For example, you can
+view the list of pods in your current project:
 
 ----
-$ oc exec -it -p <pod> -c <container> /bin/bash -c mongo
+$ oc get pods
 ----
 
-To enter a container from the host:
+Then, open a remote shell session to the desired pod:
 
 ----
-$ oc exec -it -p <pod> -c <container> /bin/bash
+$ oc rsh <pod>
 ----
 
-[IMPORTANT]
+When you enter the container, the required SCL is automatically enabled.
+
+You can now run the *mongo* command from the bash shell to start a MongoDB
+interactive session:
+
+====
+----
+bash-4.2$ mongo
+MongoDB shell version: 2.4.9
+connecting to: test
+>
+----
+====
+
+From the Mongo session, you can perform normal MongoDB operations. For example,
+to switch to the *sampledb* database and authenticate as the *admin* MongoDB user:
+
 ====
-link:https://access.redhat.com/errata/RHSA-2015:1650[For security purposes], the
-`oc exec` command does not work when accessing privileged containers. See the
-link:../cli_reference/basic_cli_operations.html#troubleshooting-and-debugging-cli-operations[CLI
-operations topic] for more information.
+----
+> use sampledb
+switched to db sampledb
+
+> db.auth("admin", "<password>") <1>
+1
+----
+<1> You can obtain this password by viewing the pod in the web console or by
+running `oc env pod/<pod_name> --list`.
 ====
 
-When you enter the container, the required software collection is automatically
-enabled.
+When you are finished, press *CTRL+D* to leave the MongoDB session.
```

Comment:
@rhcarvalho Gotcha. Added this stuff mainly per https://github.com/openshift/openshift-docs/issues/866. Auth instructions look OK to you in general?